src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,14 @@
 package core.basesyntax;
 
 public class UserService {
-    public void registerUser(User user) {
 
+    public void registerUser(User user) {
+        try {
+            new PasswordValidator().validate(user.getPassword(), user.getRepeatPassword());
+            saveUser(user);
+        } catch (PasswordValidationException passwordValidationException) {
```

Comment:
```suggestion
        } catch (PasswordValidationException e) {
```
that's a common convention to call exception 'e' in catch block